Modules/_ssl.c

```diff
@@ -4625,7 +4634,11 @@ PySSL_RAND(int len, int pseudo)
     if (bytes == NULL)
         return NULL;
     if (pseudo) {
+#ifdef OPENSSL_VERSION_1_1
+        ok = RAND_bytes((unsigned char*)PyBytes_AS_STRING(bytes), len);
```

Comment:
I don't think it is safe to replace ``RAND_pseudo_bytes`` with ``RAND_bytes``. A RAND engine may implement ``bytes`` and ``pseudobytes`` differently, e.g. ``bytes`` may not fill the buffer to the end in case of an error.

The function https://docs.python.org/3/library/ssl.html#ssl.RAND_pseudo_bytes has been deprecated. We can drop when OpenSSL 1.2.0 removes the feature.

Reply:
Thanks, I replaced it with:
```
        ok = (_PyOS_URandom((unsigned char*)PyBytes_AS_STRING(bytes), len) == 0 ? 1 : 0);
```